testing/run_test_suite.sh

```diff
@@ -80,19 +80,30 @@ FAILED_FLAKY_FILE=${TMP_REPORT_DIR}/failed_flaky
 # (will be empty if running from "master").
 FILES_CHANGED=$(git diff --name-only HEAD $(git merge-base HEAD master))
 
+# Spanner Backup tests will be skipped if this env variable is unset
+GOOGLE_SPANNER_RUN_BACKUP_TESTS=0
+
 # If the file RUN_ALL_TESTS is modified, or if we were not triggered from a Pull
 # Request, run the whole test suite.
 if [ -z "$PULL_REQUEST_NUMBER" ]; then
     RUN_ALL_TESTS=1
 else
+    labels=$(curl "https://api.github.com/repos/GoogleCloudPlatform/php-docs-samples/issues/$PULL_REQUEST_NUMBER/labels")
+    
     # Check to see if the repo includes the "kokoro:run-all" label
-    if curl "https://api.github.com/repos/GoogleCloudPlatform/php-docs-samples/issues/$PULL_REQUEST_NUMBER/labels" \
-        | grep -q "kokoro:run-all"; then
+    if  grep -q "kokoro:run-all" <<< $labels; then
         RUN_ALL_TESTS=1
     else
         RUN_ALL_TESTS=0
     fi
 
+    # Check to see if the repo includes the "spanner:run-backup-tests" label
+    if grep -q "spanner:run-backup-tests" <<< $labels; then
+        GOOGLE_SPANNER_RUN_BACKUP_TESTS=1
+    else
+        GOOGLE_SPANNER_RUN_BACKUP_TESTS=0
```

Comment:
You can remove the `else`... no need to export the variable if we don't want to run the tests. Also, use `export` so that the `TESTCMD` will read the envvar
```suggestion
        export GOOGLE_SPANNER_RUN_BACKUP_TESTS=1
```